src/FileLoader/ImageLoaders/BMPLoader.cs

```diff
@@ -1,13 +1,60 @@
 ﻿using FileLoader.Interfaces;
+using System.IO;
 using System;
+using Shared.Structs;
+using FileLoader.structs;
 
 namespace FileLoader.ImageLoaders
 {
     public class BMPLoader : IImageLoader
     {
         public IImage Load(string path)
+        {            
+            try
+            {
+                var bmpBytes = File.ReadAllBytes(path);
+                var bmpFile = new BMPFile();
+
+                var resolutions = GetBMPResolutions(bmpBytes);
+                bmpFile.Width = resolutions.Item1;
+                bmpFile.Height = resolutions.Item2;
+                bmpFile.Size = GetFileSize(bmpBytes);
+            }
+            catch
+            {
+                throw new NotImplementedException();
+            }
+            return null;
+        }
+
+        public (int, int) GetBMPResolutions(byte[] BMPBytes)
+        {
+            var widthBytes = new byte[4];
+            var heightBytes = new byte[4];
+            Array.Copy(BMPBytes,18,widthBytes,0, 4);
```

Comment:
I see that you dont have autoformatter on in Visual Studio. Please download `Format document on save` and use it